kube_apiserver_metrics/metadata.csv

```diff
@@ -6,6 +6,7 @@ kube_apiserver.go_threads,gauge,,thread,,Number of OS threads created,0, kube_ap
 kube_apiserver.go_goroutines,gauge,,,,Number of goroutines that currently exist,0, kube_apiserver, go routines
 kube_apiserver.APIServiceRegistrationController_depth,gauge,,,,Current depth of workqueue: APIServiceRegistrationController,0, kube_apiserver, service registration controller depth
 kube_apiserver.etcd_object_counts,gauge,,object,,Number of stored objects at the time of last check split by kind,0, kube_apiserver, etcd object counts
+kube_apiserver.etcd.db.total_size,gauge,,byte,,The total size of the etcd database file physically allocated in bytes.,0, kube_apiserver, etcd db total size
```

Comment:
should we mention that it requires k8s 1.19+ ?

Reply:
note that the metric is still inn alpha, I'd say let's wait for it to be stable before adding it to the integration.

Reply:
every `etcd` metrics are still in "alpha". 
I will add 2 comments to the metrics description:
* require k8s 1.19+
* can be remove at any time by kubernetes